pkg/steps/release/create_release.go

```diff
@@ -125,22 +192,31 @@ func (s *assembleReleaseStep) Provides() (api.ParameterMap, api.StepLink) {
 			} else {
 				return "", fmt.Errorf("image stream %s has no accessible image registry value", "release")
 			}
-			return fmt.Sprintf("%s:%s", registry, "latest"), nil
+			return fmt.Sprintf("%s:%s", registry, tag), nil
 		},
-	}, api.ReleasePayloadImageLink(api.PipelineImageStreamTagReference("latest"))
+	}, api.ReleasePayloadImageLink(api.PipelineImageStreamTagReference(tag))
 }
 
-func (s *assembleReleaseStep) Name() string { return "[release:latest]" }
+func (s *assembleReleaseStep) Name() string {
+	if s.latest {
+		return "[release:latest]"
+	}
+	return "[release:initial]"
+}
 
 func (s *assembleReleaseStep) Description() string {
-	return fmt.Sprintf("Create a release image in the release image stream")
+	if s.latest {
+		return "Create the release image containing all images built by this job"
+	}
+	return "Create the release image for the initial state of this job"
```

Comment:
initial state of current release at the time the job ran? could be more clear

Reply:
ok